shared/diagram.py

```diff
@@ -1,42 +1,130 @@
+    mapresources = {
+        "aws_lambda_function": "Lambda",
+        "aws_emr_cluster": "EMRCluster",
+        "aws_emr": "EMR",
+        "aws_elasticsearch_domain": "ES",
+        "aws_msk_cluster": "ManagedStreamingForKafka",
+        "aws_sqs_queue_policy": "SQS",
+        "aws_instance": "EC2",
+        "aws_eks_cluster": "EKS",
+        "aws_autoscaling_group": "AutoScaling",
+        "aws_ecs_cluster": "ECS",
+        "aws_db_instance": "RDS",
+        "aws_elasticache_cluster": "ElastiCache",
+        "aws_docdb_cluster": "DocumentDB",
+        "aws_internet_gateway": "InternetGateway",
+        "aws_nat_gateway": "NATGateway",
+        "aws_elb_classic": "ELB",
+        "aws_elb": "ELB",
+        "aws_route_table": "RouteTable",
+        "aws_subnet": "PublicSubnet",
+        "aws_network_acl": "Nacl",
+        "aws_vpc_peering_connection": "VPCPeering",
+        "aws_vpc_endpoint_gateway": "Endpoint",
+        "aws_iam_policy": "IAMPermissions",
+        "aws_iam_user": "User",
+        "aws_iam_group": "IAM",
+        "aws_iam_role": "IAMRole",
+        "aws_iam_instance_profile": "IAM",
+        "aws_efs_file_system": "EFS",
+        "aws_s3_bucket_policy": "S3",
+        "aws_media_connect": "ElementalMediaconnect",
+        "aws_media_live": "ElementalMedialive",
+        "aws_api_gateway_rest_api": "APIGateway",
+        "aws_sagemaker_notebook_instance": "Sagemaker",
+        "aws_ssm_document": "SSM",
+        "aws_cognito_identity_provider": "Cognito",
+        "aws_iot_thing": "InternetOfThings",
+        "aws_general": "General",
+        "aws_appsync_graphql_api": "Appsync",
+        "aws_iot_analytics": "IotAnalytics",
+        "aws_securityhub_account": "SecurityHub",
+        "aws_trusted_advisor": "TrustedAdvisor",
+        "aws_kinesis_firehose": "KinesisDataFirehose",
+        "aws_glue": "Glue",
+        "aws_quicksight": "Quicksight",
+        "aws_cloud9": "Cloud9",
+        "aws_organizations_account": "Organizations",
+        "aws_config": "Config",
+        "aws_auto_scaling": "AutoScaling",
+        "aws_backup": "Backup",
+        "aws_cloudtrail": "Cloudtrail",
+        "aws_cloudwatch": "Cloudwatch",
+        "aws_data_pipeline": "DataPipeline",
+        "aws_dms": "DMS",
+        "aws_elastic_beanstalk_environment": "EB",
+        "aws_fms": "FMS",
+        "aws_global_accelerator": "GAX",
+        "aws_inspector": "Inspector",
+        "aws_cloudfront_distribution": "CloudFront",
+        "aws_migration_hub": "MigrationHub",
+        "aws_sns_topic": "SNS",
+        "aws_vpc": "VPC",
+        "aws_iot": "IotCore",
+        "aws_iot_certificate": "IotCertificate",
+        "aws_iot_policy": "IotCertificate",
+        "aws_iot_type": "IotCore",
+        "aws_iot_billing_group": "IotCore",
+        "aws_iot_job": "IotJobs",
+    }
 
 
 class BaseDiagram(object):
+    def __init__(self, name: str, filename: str, engine: str = "sfdp"):
+        """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [1 blank line required between summary line and description (found 0) (D205)](https://app.codacy.com/gh/Cloud-Architects/cloud-discovery/pullRequest?prid=5655786)